elastic/kibana
# [ResponseOps][Rules] Fix case action templates in stack for security serverless

Fixes #195599

## Summary

This PR ensures that we can use templates in the case action when:
1. the project is serverless security, and
2. the rule is created in stack management

### How to test

1. Add the following line to `serverless.yml` - `xpack.cloud.serverless.project_id: test-123`
3. Start Elastic search in serverless security mode - `yarn es serverless --projectType security`
4. Start Kibana in serverless security mode - `yarn start --serverless=security`
5. Go to Security > Cases > Settings and add a template.
6. Go to stack and create a rule with the case action.
7. Confirm the template created in step 5 can be selected.

<img width="586" alt="Screenshot 2024-10-10 at 15 00 46" src="https://github.com/user-attachments/assets/5379e1d1-f0eb-4829-9604-ee5a0e3d050b">

**Please double-check also that the templates in the case action still work as expected in normal scenarios.**
